packages/fxa-content-server/app/scripts/views/mixins/password-mixin.js

```diff
@@ -214,21 +227,18 @@ export default {
 
   showPasswordHelper() {
     if (this.$('#vpassword').is(':focus')) {
-      const inverse =
-        this.$('#password-strength-balloon').css('display') === 'block'
-          ? 'none'
-          : 'block';
-      this.$('.input-help:not(.password-strength-balloon)').css(
-        'display',
-        inverse
-      );
+      const vPwBalloon = this.$('#vpassword-balloon');
+
+      if (this.$('#password-strength-balloon').hasClass('hidden')) {
+        vPwBalloon.removeClass('hidden');
+      } else {
+        vPwBalloon.addClass('hidden');
+      }
```

Comment:
Haha, it was a trip down memory lane reading some jQuery docs working on this ticket and I'd forgotten about `toggleClass`. I don't think we want to use that here though since we're checking for if element A has a class then element B should always remove a class else it should always add the class - not sure toggling the class is what we want in case that gets out of sync or something. 🤔